portal/src/main/webapp/js/src/oncoprint/setup.js

```diff
@@ -871,6 +868,14 @@ window.CreateCBioPortalOncoprintWithToolbar = function (ctr_selector, toolbar_se
 	    oncoprint.setSortConfig({'type':'order', 'order':order});
 	};
 	
+	var updateViewMenuLabels = function(state) {
```

Comment:
Sure, if the pattern used here prevents people from considering to call functions before this block has defined them all, it's not important.